src/shared/validation/index.ts

```diff
@@ -25,3 +25,15 @@ export const validate = (schema: ObjectSchema<any> | ArraySchema<any>, state: an
         return false;
     }
 };
+
+export function checkIfFilesAreCorrectType(files?: [File]): boolean {
```

Comment:
I guess this should be called checkIfFilesAreDocx or similar, given the type is not configurable